Test/Integration/Model/Transaction/OrderTest.php

```diff
@@ -15,172 +15,128 @@
+use Taxjar\SalesTax\Model\Transaction\Order as TaxjarOrder;
+use Taxjar\SalesTax\Test\Integration\IntegrationTestCase;
+use Taxjar\SalesTax\Util\Fixtures\Catalog\ProductBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Customer\AddressBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Customer\CustomerBuilder;
+use Taxjar\SalesTax\Util\Fixtures\Sales\OrderBuilder;
 
 /**
- * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
- * @see https://app.hiptest.com/projects/69435/test-plan/folders/419534/scenarios/2587535
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)/**
+ *
+ * @magentoDbIsolation enabled
+ * @magentoAppIsolation enabled
  */
-class OrderTest extends \PHPUnit\Framework\TestCase
+class OrderTest extends IntegrationTestCase
 {
     /**
-     * @var CleanupReservationsInterface
-     */
-    protected $cleanupReservations;
-
-    /**
-     * @var Order
+     * @var TaxjarOrder
      */
-    protected $order;
-
-    /**
-     * @var \Taxjar\SalesTax\Model\Transaction\Order
-     */
-    protected $transactionOrder;
+    protected $taxjarOrder;
 
     protected function setUp(): void
     {
-        $this->cleanupReservations = Bootstrap::getObjectManager()->get(CleanupReservationsInterface::class);
-        $this->order = Bootstrap::getObjectManager()->get(Order::class);
-        $this->transactionOrder = Bootstrap::getObjectManager()->get('Taxjar\SalesTax\Model\Transaction\Order');
+        $this->taxjarOrder = $this->objectManager->get(TaxjarOrder::class);
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple.php
-     */
     public function testDefaultOrder()
     {
-        $order = $this->order->loadByIncrementId('100000002');
-        $result = $this->transactionOrder->build($order);
+        $order = OrderBuilder::anOrder()->build();
+        $result = $this->taxjarOrder->build($order);
 
         $this->assertEquals('api', $result['provider'], 'Invalid provider');
-        $this->assertEquals(68, $result['amount'], 'Invalid total amount');
-        $this->assertEquals(0, $result['shipping'], 'Invalid shipping amount');
+        $this->assertEquals(45, $result['amount'], 'Invalid total amount');
+        $this->assertEquals(15, $result['shipping'], 'Invalid shipping amount');
         $this->assertEquals(0, $result['sales_tax'], 'Invalid sales tax amount');
     }
 
-    /**
-     * @magentoDataFixture ../../../../app/code/Taxjar/SalesTax/Test/Integration/_files/transaction/order_simple.php
-     */
     public function testPositiveDecimals()
     {
-        $order = $this->order->loadByIncrementId('100000002');
-        $result = $this->transactionOrder->build($order);
+        $order = OrderBuilder::anOrder()->build();
+        $result = $this->taxjarOrder->build($order);
 
-        $this->assertEquals(27.0, $result['line_items'][0]['unit_price'], 'Invalid provider');
+        $this->assertEquals(10.0, $result['line_items'][0]['unit_price'], 'Invalid provider');
```

Comment:
The message given if this test fails needs updated. 'Invalid provider' would be a misleading message if this test failed.